### ISSUE TITLE
Quality adjusted alignment crashes non-deterministically

### ISSUE THREAD

**Issue by jeizenga** [vgteam/vg#1089]
I've verified that this happens in both map and mpmap, running multithreaded or single threaded. The crash happens in the gssw traceback. I currently suspect the gssw refactor that enabled score-only alignments.

**Comment by adamnovak** [comment]
mpmap never does the new score-only alignments, right?

Also, is there a particular dataset you have successfully replicated the problem with?

How do you know the problem happens during the traceback?

**Comment by jeizenga** [comment]
Correct, no score-only alignments in mpmap. I know the crash is during the traceback because gssw throws an error indicating that it's doing a traceback. The actual bug might very well be during DP. I don't really know.

**Comment by adamnovak** [comment]
@jeizenga Did you ever manage to make this happen again?

**Comment by jeizenga** [comment]
No, I didn't. I'm ready to call this a build mismatch to a dependency or something weird like that until it happens again.

**Comment by jeizenga** [comment]
I've seen this happen again. It's rare, but it does still happen.

**Comment by jeizenga** [comment]
Actually, it's common enough that it will usually happen for me on a run of 100,000 reads.

**Comment by adamnovak** [comment]
Can you make it happen under gdb? And get a stack trace?
